### ISSUE TITLE
Show ;go2 room ID

### ISSUE THREAD

**Issue by rpherbig** [rcuhljr/drshops#2]
Lich users really enjoy having access to that :)

Is there an equivalent for other frontends?

I don't think should be a new column; maybe a popover on Owner? Maybe Owner and Store could be combined in the same way?

**Comment by rcuhljr** [comment]
Good point, I planned on adding a 'directions' blurb for each store with the lich room number, room name, genie room number if we get our hands on those.

**Comment by rpherbig** [comment]
Maybe it's a separate issue (on mobile, can't make a new one right now),
but it would be awesome to let users submit room IDs from the site.
